pkg/ccl/benchccl/rttanalysisccl/testdata/benchmark_expectations

```diff
@@ -1,19 +1,19 @@
 exp,benchmark
+23,AlterPrimaryRegion/alter_empty_database_alter_primary_region
+24,AlterPrimaryRegion/alter_empty_database_set_initial_primary_region
+23,AlterPrimaryRegion/alter_populated_database_alter_primary_region
+25,AlterPrimaryRegion/alter_populated_database_set_initial_primary_region
 22,AlterRegions/alter_empty_database_add_region
 23,AlterRegions/alter_empty_database_drop_region
 22,AlterRegions/alter_populated_database_add_region
 23,AlterRegions/alter_populated_database_drop_region
-25,AlterPrimaryRegion/alter_empty_database_set_initial_primary_region
-25,AlterPrimaryRegion/alter_empty_database_alter_primary_region
-26,AlterPrimaryRegion/alter_populated_database_set_initial_primary_region
-25,AlterPrimaryRegion/alter_populated_database_alter_primary_region
-25,AlterSurvivalGoals/alter_empty_database_from_zone_to_region
-25,AlterSurvivalGoals/alter_empty_database_from_region_to_zone
-65,AlterSurvivalGoals/alter_populated_database_from_zone_to_region
-65,AlterSurvivalGoals/alter_populated_database_from_region_to_zone
-28,AlterTableLocality/alter_from_global_to_regional_by_table
-28,AlterTableLocality/alter_from_regional_by_table_to_global
+23,AlterSurvivalGoals/alter_empty_database_from_region_to_zone
+23,AlterSurvivalGoals/alter_empty_database_from_zone_to_region
+43,AlterSurvivalGoals/alter_populated_database_from_region_to_zone
+43,AlterSurvivalGoals/alter_populated_database_from_zone_to_region
 24,AlterTableLocality/alter_from_global_to_rbr
-24,AlterTableLocality/alter_from_regional_by_table_to_rbr
-22,AlterTableLocality/alter_from_rbr_to_regional_by_table
+26,AlterTableLocality/alter_from_global_to_regional_by_table
 22,AlterTableLocality/alter_from_rbr_to_global
+22,AlterTableLocality/alter_from_rbr_to_regional_by_table
+26,AlterTableLocality/alter_from_regional_by_table_to_global
+24,AlterTableLocality/alter_from_regional_by_table_to_rbr
```

Comment:
This diff is hard to read because these lines were re-ordered. I expect to have to rebase once https://github.com/cockroachdb/cockroach/pull/69020 is merged.

Reply:
Done